src/index.js

```diff
@@ -7,15 +7,18 @@ class TelegramLoginButton extends React.Component {
     super(props);
   }
   componentDidMount() {
-    const { botName, dataSize, requestAccess, usePic, dataOnauth } = this.props;
+    const { botName, dataSize, dataRadius, requestAccess, usePic, dataOnauth } = this.props;
```

Comment:
Can you also help refactor follow params here?/
`['size', 'userpic', 'init_auth', 'request_access', 'radius', 'min_width', 'max_width'];`